src/CRUD.js

```diff
@@ -0,0 +1,147 @@
+const newTask = document.querySelector('.task-input');
+const add = document.querySelector('.add');
+const list = document.querySelector('.lists');
+const reload = document.querySelector('.fa-arrows-rotate');
+
+const completed = false; let
+  index = 1;
+let listArr = [];
+
+const showTasks = () => {
+  const storedData = localStorage.getItem('data');
+  // index = 0;
```

Comment:
- [ ] Kindly remove all commented-out un used/ unwanted pieces of code from your code base. It makes it look messy and cluttered.